LoopKitUI/Views/Information Screens/DeliveryLimitsInformationView.swift

```diff
@@ -59,8 +59,11 @@ public struct DeliveryLimitsInformationView: View {
         VStack(alignment: .leading, spacing: 20) {
             Text(DeliveryLimits.Setting.maximumBolus.title)
             .font(.headline)
-            Text(LocalizedString("Maximum Bolus is the highest bolus amount that you will allow \(appName) to recommend at one time to cover carbs or bring down high glucose.", comment: "Information about maximum bolus"))
-            .foregroundColor(.secondary)
+            VStack(alignment: .leading, spacing: 20) {
+                Text(LocalizedString("Maximum Bolus is the highest bolus amount \(appName) will recommend, or allow you to deliver, at one time to cover carbs or bring down high glucose.", comment: "Information about maximum bolus"))
+                Text(LocalizedString("Your maximum bolus will also set a cap on how much insulin will be allowed for automatic dosing.  \(appName) will restrict automatic dosing to amounts that will not cause your insulin on board to exceed twice your maximum bolus.", comment: "Information about maximum sutomated insulin on board"))
```

Comment:
Small typo in the translation comment: "sutomated"

Reply:
Fixed and tested. (Had to update my Xcode to test it - for some reason, my Xcode had decided to stop working.)
In addition to the typo fix, I changed the format for first line under basal so all the basal and bolus lines have similar formatting.
If you don't like that style, I'll modify it.